tests/post_training/data/ptq_reference_data.yaml

```diff
@@ -22,6 +22,16 @@ hf/hf-internal-testing/tiny-random-gpt2_backend_OV:
   metric_value: null
 hf/hf-internal-testing/tiny-random-gpt2_backend_TORCH:
   metric_value: null
+torchvision/resnet18_backend_FP32:
```

Comment:
What differ between torchvision and timm models?
If it's same but timm can not be exported to FX, timm/resnet18 can be removed

Reply:
I've removed timm/resnet18 and replaced with torchvision/resnet18